AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/MutationQueue/OutgoingMutationQueueTestsWithMockStateMachine.swift

```diff
@@ -65,33 +65,58 @@ class OutgoingMutationQueueMockStateTest: XCTestCase {
         waitForExpectations(timeout: 1)
     }
 
-    func testRequestingEvent_subscriptionSetup() {
+    func testRequestingEvent_subscriptionSetup() throws {
         let semaphore = DispatchSemaphore(value: 0)
         stateMachine.pushExpectActionCriteria { action in
             XCTAssertEqual(action, OutgoingMutationQueue.Action.receivedSubscription)
             semaphore.signal()
         }
         stateMachine.state = .starting(apiBehavior, publisher)
         semaphore.wait()
+
+        let json = "{\"id\":\"1234\",\"title\":\"t\",\"content\":\"c\",\"createdAt\":\"2020-09-03T22:55:13.424Z\"}"
```

Comment:
I changed `json` from "{}" to a decodable one otherwise the test will fail because it returns earlier from https://github.com/aws-amplify/amplify-ios/pull/759/files#diff-574bec15c64b358883e284ae61f479e1R195